docs/storage_v2/built_in_features/reward.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
do not get attached to the drive state, because it is more from the previous implementation